src/software/simulated_tests/validation_functions/touch_ball_in_enemy_defense_validation.cpp

```diff
@@ -0,0 +1,37 @@
+#include "software/simulated_tests/validation_functions/touch_ball_in_enemy_defense_validation.h"
+
+#include "software/new_geom/util/contains.h"
+#include "software/new_geom/util/intersects.h"
+
+void robotTouchedBallInEnemyDefense(std::shared_ptr<World> world_ptr,
```

Comment:
may want to specify that no _friendly_ robot touches the ball in the enemy defense